release-process.md

```diff
@@ -319,4 +319,18 @@ Once everything is working (website docs, website changes) create an announcemen
 and then send an e-mail to the mailing list. To create an announcement, create a post under
 `news/_posts` and then run `jekyll build`.
 
+<h4>Update a test suite depending on old releases</h4>
+
+Since Spark always keeps the latest maintance released of each branch in the mirror network,
+we need to update a test suite to check backward compatibility with downloading old versions.
+For now, there is one test suite, `HiveExternalCatalogVersionsSuite`.
+
+For a new maintenance release, `HiveExternalCatalogVersionsSuite` will not fail.
+However, it will cause many trials and fallbacks in all Jenkins builds over corresponding branches.
+This updates should be done as a separate PR after release announcement. (See SPARK-25570)
```

Comment:
I think the above description is not needed. Just explain what the release managers need to do. This is Spark SQL specific. I am afraid this might confuse the release managers who are not familiar with Spark SQL.